### ISSUE TITLE
io.write _undefined_

### ISSUE THREAD

**Issue by lacdael** [orbitdb/orbitdb#1014]
A fresh react install with minimal code, but I cannot create a db. stepping through the error it shows that io.write in ipfs-access-controller.js is undefined. io.read is defined, but io.write isn't.

```
IPFSAccessController.save ERROR: TypeError: io.write is not a function
    at IPFSAccessController.save (ipfs-access-controller.js:46:1)
    at AccessControllers.create (access-controllers.js:68:1)
    at async OrbitDB._determineAddress (OrbitDB.js:336:1)
    at async OrbitDB.create (OrbitDB.js:359:1)
    at async App.js:38:1
```

versions :
-    "ipfs": "^0.63.5",
-    "orbit-db": "^0.28.6",

```
  const ipfs = await create(
  {
    repo: nodeId,
  }
  )
   const anId = await ipfs.id();
   const orbitdb = await OrbitDB.createInstance( ipfs ,{
    id : 'an-id',
  } );

const db = await orbitdb.create("test","keyvalue", {
        overwrite: true,
         replicate:false,
         accessController:{
             write:["*"]
         }
     });
```

I tried to run https://github.com/orbitdb/orbit-db-control-center, too, but it fails on a migration check. Any guidance to get a working setup would be greatly appreciated.

**Comment by Jon-Biz** [comment]
The write and read functions in question are defined here:

https://github.com/orbitdb/orbit-db-io/blob/main/index.js

`src/utils/io` passes back the read function wrapped in async function, and passes the write function back directly.

**Comment by inspiraluna** [comment]
I had this problem too, i think fixed it by downgrading  orbit-db-io@1.0.2 . 
Make sure its not using accidently a new version.

**Comment by haydenyoung** [comment]
References an older version of OrbitDB. You can access the latest version using `npm i @orbitdb/core`.
